### DISCUSSION TITLE
Modify style components on icons

### DISCUSSION THREAD

**Discussion by bvsillorequez** [twilio-labs/paste#948] [Q&A] [answered]
I am trying to modify the color and size of an icon but the color I am looking for isn't available for text colors and same with the size.  Is there a way to modify them like you can with <Text> by adding a style={{}} component?

**Comment by TheSisb** [comment]
Hi @bvsillorequez,

Our icons only accept token values as props so they remain consistent within Paste applications. Can you share some additional context as to why you're looking to use non-token values?

**Comment by bvsillorequez** [comment]
I would like to have the color of the beta tag in console to be the same as $color-background-user since it is the same color as our avatar.  Currently there isn't a color similar to that in the text colors.

**Comment by serifluous** [comment]
Hey @bvsillorequez ! You can use $color-text-new for the beta tag (which is also more appropriately named for your use case) and is accessible on our background colors.

**Reply by bvsillorequez** [reply]
I am using $color-text-new for the sidebar nav but that color is too dark to be in the masthead.  We need something lighter like $color-background-user so it doesn't blend in.

**Reply by serifluous** [reply]
got it—Can you share a screenshot or mockup of what you're trying to achieve?

**Reply by bvsillorequez** [reply]
<img width="216" alt="Screen Shot 2020-11-17 at 12 32 34 PM" src="https://user-images.githubusercontent.com/60529554/99444433-0f9aa100-28d1-11eb-8bfb-a8f0a6922a6a.png">
This shows $color-text-new for both beta tags. The tag in the masthead we want it to be the color of the avatar
<img width="112" alt="Screen Shot 2020-11-17 at 12 32 38 PM" src="https://user-images.githubusercontent.com/60529554/99444500-25a86180-28d1-11eb-8962-9842832310f0.png">

**Reply by serifluous** [reply]
Cool, thanks! Makes sense. I'll sync with our team and add in a new text token for you (or at least provide an alternate design solution).

**Reply by bvsillorequez** [reply]
Great!  Thank you!

**Comment by serifluous** [comment, marked as answer]
Chatted with the team: Since Console will be in beta just for the short-term, we're suggesting that y'all use the white text color (`color-text-inverse`) and the full text can be something like "Console (beta)" or "Console – beta". A light purple text color might not also be meaningfully different from white text when they're next to each other.

@johannaweintraub lemme know if that solution works for you ^

**Reply by johannaweintraub** [reply]
This also affects admin center masthead but for now, let's just leave it as is.

**Reply by johannaweintraub** [reply]
We're using the $color-text-icon-inverse
